chaos_genius/views/kpi_view.py

```diff
@@ -308,8 +328,29 @@ def edit_kpi(kpi_id):
             dashboard_id_list = list(set(dashboard_id_list))
 
             for key, value in data.items():
+                print(key, value)
+                if key not in do_not_run_analytics_list:
+                    run_analytics = True
                 if chech_editable_field(meta_info, key):
                     setattr(kpi_obj, key, value)
+            if run_analytics:
```

Comment:
Can we have a log here like "Re-running anomaly and deepdrills since KPI was edited"?

Reply:
Added in 5cf098efefcd4234b56efb26df8375e34682e703